bliss/encoder.py

```diff
@@ -330,6 +335,23 @@ def _get_loss(self, pred: Dict[str, Distribution], true_tile_cat: TileCatalog):
         return loss_with_components
 
     def _generic_step(self, batch, logging_name, log_metrics=False, plot_images=False):
+        do_data_augmentation = self.data_augmentation.get("do_data_augmentation")
```

Comment:
We only want to do data augmentation during training, never during validation or testing. Therefore, could we move the data augmentation call to `training_step(...)` rather than here?